typescript/libs/view_helpers.py

```diff
@@ -46,33 +47,35 @@ def get_info(view):
         file_name = view.file_name()
         if is_typescript(view):
             info = _file_map.get(file_name)
-            if not info:
-                info = FileInfo(file_name, None)
-                info.view = view
-                info.client_info = cli.get_or_add_file(file_name)
-                set_file_prefs(view)
-                _file_map[file_name] = info
-                # Open the file on the server
-                open_file(view)
-                if view.is_dirty():
-                    if not view.is_loading():
-                        reload_buffer(view, info.client_info)
+            if open_if_not_found:
+                if not info or info.is_open is False:
```

Comment:
The catch is that I want to avoid doing anything on the worker server either if the ``open_if_not_cached`` is False (line 69). The situation applies when I close a file and call ``get_info`` in the  ``on_activated`` function.